### PR TITLE
Feature/added file processing log printroom

### PR DESCRIPTION
**Pull request by FurkanKayar55**
## Link to JIRA ticket

Add a link to the JIRA ticket that the changes in this PR describe.

### *What is the problem we're trying to solve*

Extend file process log endpoint with file type

### *What changes have we introduced*

Extend file process log endpoint with file type

Include any links to commits that introduce significant additions of code if they help explain the process of coming to the solution. e.g Addition of test database setup, addition of first test and production class, removal of buggy code, etc.

#### _Checklist_

- [ ] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [ ] Added tests to cover all new production code
- [ ] Added comments to the README or updated relevant documentation _(add link to documentation)_, where necessary.
- [ ] Checked all code for possible refactoring
- [ ] Code pipeline builds correctly

### *Follow up actions after merging PR*

Are there any next steps that need to addressed after merging this PR? Add them here.

### CODE REVIEW

File: ChargesApi/V1/Gateways/Services/AwsS3FileService.cs [hunk 1]
```diff
@@ -121,7 +137,8 @@ private async Task<(string year, string fileStatus, string valuesType)> GetObjec
             var year = taggingResponse.Tagging.Where(t => t.Key == "year").Select(t => t.Value).FirstOrDefault();
             var fileStatus = taggingResponse.Tagging.Where(t => t.Key == "status").Select(t => t.Value).FirstOrDefault();
             var valuesType = taggingResponse.Tagging.Where(t => t.Key == "valuesType").Select(t => t.Value).FirstOrDefault();
-            return (year, fileStatus, valuesType);
+            var fileType = taggingResponse.Tagging.Where(t => t.Key == "fileType").Select(t => t.Value).FirstOrDefault();
```
Comment (sandipchandra) [inline ChargesApi/V1/Gateways/Services/AwsS3FileService.cs:140]: If you are using file type tag to retrieve the value , you need to make changes when the Estimate file is uploaded ,  create a new tag there called file type and also when you are generating the file, you also need to create the same file tag key with different value

**Review by sandipchandra — CHANGES_REQUESTED** [review]
Please review the comments, thanks.

File: ChargesApi/V1/Gateways/Services/AwsS3FileService.cs (hunk 1 above)
Comment (FurkanKayar55) [reply ChargesApi/V1/Gateways/Services/AwsS3FileService.cs:144]: Done

File: ChargesApi/V1/Domain/Enums.cs [hunk 2]
```diff
@@ -59,4 +59,10 @@ public enum ChargeSubGroup
         Estimate,
         Actual
     }
+    [JsonConverter(typeof(JsonStringEnumConverter))]
+    public enum FileType
+    {
+        Estimate,
```
Comment (sandipchandra) [inline ChargesApi/V1/Domain/Enums.cs:65]: This could be EstimateOrActual

File: ChargesApi/V1/Gateways/Services/AwsS3FileService.cs [hunk 3]
```diff
@@ -29,7 +30,11 @@ public async Task<FileLocationResponse> UploadFile(IFormFile formFile, string fi
             var location = $"uploads/{fileName}";
             using (var stream = formFile.OpenReadStream())
             {
-                var tagSet = new List<Tag> { new Tag { Key = "status", Value = "Uploaded" } };
+                var tagSet = new List<Tag>
+                {
+                    new Tag { Key = "status", Value = "Uploaded" },
+                    new Tag { Key = "fileType", Value = "Estimate"}
```
Comment (sandipchandra) [inline ChargesApi/V1/Gateways/Services/AwsS3FileService.cs:36]: This also needs to be in sync with EstimateOrActual

**Review by sandipchandra — CHANGES_REQUESTED** [review]

File: ChargesApi/V1/Gateways/Services/AwsS3FileService.cs (hunk 3 above)
Comment (FurkanKayar55) [reply ChargesApi/V1/Gateways/Services/AwsS3FileService.cs]: Done

**Review by sandipchandra — APPROVED** [review]

**Pull request merged**